trappy/utils.py

```diff
@@ -75,7 +75,7 @@ def handle_duplicate_index(data,
     index = data.index
     new_index = index.values
 
-    dups = index.get_duplicates()
+    dups = index[index.duplicated()].unique()
```

Comment:
`unique()` was introduced in pandas 0.23.0.  Can you bump the pandas version requirement in `setup.py` to 0.23.0 as part of this commit please?

Reply:
Will do!

Reply:
Mmm wouldn't 0.15.0 be enough? I have nothing against 0.23.0, but looking at the API doc it seems that:
* `Index.unique()` was introduced in 0.13.0 (https://pandas.pydata.org/pandas-docs/version/0.13/generated/pandas.Index.unique.html)
* `Index.duplicated()` was introduced in 0.15.0 (https://pandas.pydata.org/pandas-docs/version/0.15/generated/pandas.Index.duplicated.html)

Reply:
Ah, right, I misread the documentation.  The `level` parameter is the one that was added in 0.23.0.  You are correct, 0.15.0 is enough.

Reply:
Do we want to take the opportunity to bump up to a higher version though? 0.15.0 is from 2014, which is getting old.

Reply:
We are setting the lower bound (`pandas>=0.15.0`), no need to bump it to something newer without a reason.

Reply:
Okay, works for me